### DISCUSSION TITLE
Simple rule

### DISCUSSION THREAD

**Discussion by renemkr** [Python-Fuzzylogic/fuzzylogic#26] [Q&A] [answered]
I'm triying to make a simple rule IF A then B
but if I write
R1 = Rule({A: B})
R1 = Rule({temp.hot: motor.fast})

Gives me an error
TypeError: 'Set' object is not iterable

**Comment by amogorkon** [comment, marked as answer]
Hi, did you figure it out?

Rules are there to map sets of one domain to a single set of another, so it expects the left side to be an iterable like a tuple.

Try

`R1 = Rule({(temp.hot, ): motor.fast})`

That should do the trick.

**Reply by renemkr** [reply]
I thought it was a simple thing, but my Python is a bit basic at the moment. Thank you very much!!
